Toggl.Core.UI/ViewModels/Settings/SettingsViewModel.cs

```diff
@@ -52,7 +52,9 @@ public sealed class SettingsViewModel : ViewModel
         private IThreadSafePreferences currentPreferences;
 
         public string Title { get; private set; } = Resources.Settings;
-        public string Version => $"{platformInfo.Version} ({platformInfo.BuildNumber})";
+        public string Version => $"{platformInfo.Version} ({platformInfo.BuildNumber}) {SyncManagerDesignation}";
```

Comment:
As agreed internally:

```suggestion
        public string Version => $"{platformInfo.Version} ({platformInfo.BuildNumber}/{syncManager.Version})";
```